source/_includes/asides/component_navigation.html

```diff
@@ -15,7 +15,13 @@
   </div>
 
   <div class='section'>
-    The {{ page.name | default: page.title }} {{ page.ha_integration_type | default: "integration" }} was introduced in Home Assistant {{ page.ha_release | default: "unknown" }},
+    The {{ page.name | default: page.title }} {{ page.ha_integration_type | default: "integration" }} was introduced in Home Assistant 
+    {% if ha_release == "pre 0.7" %}
+    <a href="/integrations/#version/0.7">{{ page.ha_release | default: "unknown" }}</a>,
+    {% else %}
+    <a href="/integrations/#version/{{page.ha_release}}">{{ page.ha_release | default: "unknown" }}</a>,
```

Comment:
I am not understanding the reason to link this? What would be the added benefit?


If I look at, for example, the ESPHome integration page, why would I click on the version to view all others introduced in that release? What is the relation/added value for the user?

Reply:
We already provide it from the main integration page as a filter. I think its odd, as the release itself, this specific integration has nothing to do with other integration in the same release.

I would opt to not to add it for that reason.

Reply:
Alright, I'll remove that link